### ISSUE TITLE
[Bug] The tool is not saved when switch to other hanging protocols.

### ISSUE THREAD

**Issue by m00n620** [OHIF/Viewers#3564]
### Describe the Bug

The previous tool state is not saved if we switch to other hanging protocols like MPR.

### Steps to Reproduce

1. Select any tool like Zoom or Pan from the tool bar.
2. Activate the MPR mode.

### The current behavior

The WindowLevelTool is active.

### The expected behavior

The previous tool(Zoom or Pan) should be active.

### OS

Windows 11

### Node version

16.18.0

### Browser

Chrome 115.0.5790.110 (Official Build) (64-bit)

**Comment by sedghi** [comment]
Yeah, I understand this and like this to be fixed in the next version, it is very annoying
